### ISSUE TITLE
arm64 == aarch64

### ISSUE THREAD

**Issue by carlocorradini** [vscode-shellcheck/shellcheck-binaries#7]
Do you want to rename Darwin 'arm64' to 'aarch64' because they are the same thing? Or keep them separated?

**Comment by carlocorradini** [comment]
The update is super easy.

From:
```sh
archive "$DARWIN_ARM64_DIR" "darwin" "arm64"
```

To:
```sh
archive "$DARWIN_ARM64_DIR" "darwin" "aarch64"
```

**Comment by felipecrs** [comment]
Perhaps it would be better to simply use Node's naming:

- https://nodejs.org/api/os.html#osplatform
- https://nodejs.org/api/os.html#osarch

**Comment by felipecrs** [comment]
Yes, we should do that.

**Comment by carlocorradini** [comment]
Fixed in PR #9
